core/lib/spree/core/number_generator.rb

```diff
@@ -0,0 +1,46 @@
+module Spree
+  module Core
+    class NumberGenerator < Module
+
+      RAND           = Random.new
+      BASE           = 10
+      DEFAULT_LENGTH = 9
+      NUMBERS        = (0..9).to_a.freeze
+      LETTERS        = ('A'..'Z').to_a.freeze
```

Comment:
@mbj I agree I prefer reading this way as well if you want to submit a PR on that..